_book/04-advanced-topics/04-controlling-the-svg-output.md

```diff
@@ -1,11 +1,128 @@
+      test-suite: score/score-013.mei
+      options:
+        adjustPageHeight: False
+        breaks: 'auto'
+        pageHeight: 800
+        pageWidth: 1800
+        scale: 100
+        scaleToPageSize: True
+        spacingStaff: 12
+
+    - name: 05-responsive-page
+      test-suite: score/score-013.mei
+      options:
+        adjustPageHeight: False
+        breaks: 'auto'
+        pageHeight: 800
+        pageWidth: 1800
+        scale: 30
+        scaleToPageSize: True
+        spacingStaff: 12
 ---
 
-[in preparation]
+### Page units and dimensions
+
+Verovio uses an abstract unit for specifying the page dimensions. By default, the page height is `2970` and the page width is `2100`. These are equivalent to the dimension of an A4 page in tenths of a millimeter. When generating SVG, these units are interpreted as pixels, which means that the default SVG image size is **2970px** height by **2100px** width. 
+
+The example below shows an empty page with the default dimensions – and the option `--justifyVertically` turned on.
+
+{% include music-notation-only example="01-default" %}
+
+The option `--mm-output` can be turned on to have the page dimension being interpreted as millimeters. In that case, the SVG image will have a size of **297mm** by **210mm**, which is recommended if the SVG is meant to be printed or converted to PDF.
+
+Page margins (`--page-margin-bottom`,  `--page-margin-left`,  `--page-margin-right` and  `--page-margin-top`) are all specified in abstract units, with a default value of `50`. That is **50px** with the default SVG output and **5mm** with the `--mm-output` option turned on.
+
+#### MEI unit
+
+Most of the options in Verovio are given in MEI units. An MEI unit (or MEI virtual unit) corresponds to half the distance between adjacent staff lines where the interline space is measured from the middle of a staff line. The value of the MEI unit in Verovio will determine the size of the staff on a page. By default, the MEI unit is `9.0`, which means that a staff space is 9 abstract units. In terms of staff size (or raster), it means a staff size of **72px** in the default SVG output, or **7.2mm** with the `--mm-output` option turned on.
+ 
+In traditional music engraving, the staff size corresponds to the raster, which would be chosen depending of the type of score to be engraved. That would mean changing the MEI unit. However, in digital environments featuring responsive layout, there should be no need to change the default value of the MEI unit. The layout should be controlled with the page (i.e., the window or screen) size and scaling (see below). Nonetheless, the unit can be adjusted in Verovio with the `--unit` option. That can be useful if Verovio is used to replicate a print layout with a specific traditional page and staff size. The table below gives an indication of values for the MEI unit in Verovio corresponding to raster sizes as found in the literature or some music notation software applications.
```

Comment:
I would move this paragraph to be the first one of this section (under the "MEI unit" heading). The last sentence can be kept here.

Reply:
Then we need to rewrite them because the MEI unit needs to be defined.

Reply:
```suggestion
In traditional music engraving, the staff size corresponds to the raster which would be chosen depending of the type and size of score to be engraved. However, in digital environments there is generally no need to change the size of the raster since the size of the notation can be changed on the screen depending on the size and orientation of the screen (i.e., "responsive" environments). This is usually changed by adjusting the page size and scaling factors (see below).

However, for some cases it may be necessary to change the size of the raster size within a digital environment. This may be useful, for example, to replicate a print layout with a specific traditional page and staff size. With Verovio, the raster size can be adjusted with the `--unit` option, which adjusts the definition of an MEI units. One MEI unit (or MEI virtual unit) corresponds to half the distance between adjacent staff lines where the interline space is measured from the middle of a staff line. 

The default value of `unit` in Verovio is `9.0`, which corresponds to 9px per unit in SVG, or 0.9mm with the `--mm-output` option enabled. The table below gives an indication of values for the MEI unit in Verovio corresponding to raster sizes as found in the literature or some music notation software applications.
```

Reply:
I've combined this paragraph and the previous one, and tried to get the definition of "unit" right.